bots/mlbridge/build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
     implementation project(':webrev')
     implementation project(':version')
     implementation project(':metrics')
+    implementation project(':bots:common')
+    implementation project(path: ':jbs')
```

Comment:
What does the `path:` prefix mean here? If it's not significant, better stay with the same pattern as the other dependencies.

Reply:
Yes sure, I removed it to be consistent.